modules/core/arrow-core/src/main/kotlin/arrow/core/Option.kt

```diff
@@ -211,3 +211,6 @@ fun <A> Boolean.maybe(f: () -> A): Option<A> =
 fun <A> A.some(): Option<A> = Some(this)
 
 fun <A> none(): Option<A> = None
+
+@Suppress("NOTHING_TO_INLINE")
+inline infix fun <A, B> OptionOf<(A) -> B>.ap(fa: OptionOf<A>): Option<B> = fa.fix().apPipe(this)
```

Comment:
In 0.8.0 all these functions will be automatically exported because `ap` appears in `Applicative`, so all instances of Applicative will get auto exports of package level extension functions that delegate to the instance. Nothing wrong in having them here for people that include the `data` module but not the `typeclass` one.

Reply:
If that is the case, is there really a need for them to be here?
I doubt someone not using typeclasses would find them useful.

Reply:
no need, they can be removed from here since the projection will add this exact same function in the type class syntax packages